src/libData/BlockData/BlockHeader/TxBlockHeader.h

```diff
@@ -35,6 +35,7 @@ class TxBlockHeader : public BlockHeaderBase {
   uint32_t m_version;
   boost::multiprecision::uint256_t m_gasLimit;
   boost::multiprecision::uint256_t m_gasUsed;
+  boost::multiprecision::uint256_t m_rewards;
```

Comment:
Is reward need to be such a big range? Is 64 bit int can fulfill already?

Reply:
the rewards = gasUsed * gasPrice, as both these 2 params are all u256, this should remain unchanged